File: src/network/model/Airly/MeasurementsAirlyResponse.java
package network.model.Airly;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class MeasurementsAirlyResponse {
    private MeasurementAirly current;
    private MeasurementAirly[] history;
    private MeasurementAirly[] forecast;

    public MeasurementAirly getCurrent() {
        return current;
    }

    public void setCurrent(MeasurementAirly current) {
        this.current = current;
    }

    public MeasurementAirly[] getHistory() {
        return history;
    }

    public void setHistory(MeasurementAirly[] history) {
        this.history = history;
    }

    public MeasurementAirly[] getForecast() {
        return forecast;
    }

    public void setForecast(MeasurementAirly[] forecast) {
        this.forecast = forecast;
    }
}

File: src/network/model/Airly/MeasurementsAirlyResponse.kt
package network.model.Airly

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class MeasurementsAirlyResponse {
    var current: MeasurementAirly? = null
    var history: List<MeasurementAirly>? = null
    var forecast: List<MeasurementAirly>? = null
}
